### DISCUSSION TITLE
Run Multiple Fastq files in DNA mode

### DISCUSSION THREAD

**Discussion by thelingxichen** [milaboratory/mixcr#1598] [Q&A] [answered]
Hi,

I want to try mixcr on tumor WGS data, but I have multiples fastq files (different run) for one patient.
Like:   Run1_fq1.gz  Run2_fq1.gz  and Run1_fq2.gz  Run2_fq2.gz  
How can I run mixcr without merging the fastq files?

Thanks,
Lingxi

**Comment by dbolotin** [comment]
You can try [process substitution](https://en.wikipedia.org/wiki/Process_substitution).

E.g.:

```
mixcr align -s hs \
    <(gzcat Run1_fq1.gz Run2_fq1.gz) \
    <(gzcat Run1_fq2.gz Run2_fq2.gz) \
    output.vdjca
```
or `zcat` in case of linux

Beware that MiXCR can't correctly calculate input size, so progress reporting will be broken, still analysis by itself will not be affected.

**Comment by dbolotin** [comment]
Please also see [docs](https://mixcr.readthedocs.io/en/master/analyze.html#analysis-of-non-enriched-or-random-fragments) for WGS analysis pipeline.

**Comment by malonzm1** [comment]
Hi,

When I run the following:

```
mixcr analyze 10x-sc-5gex -s hsa --rna --threads 4 --force-overwrite <(zcat SRR7159836/fastq/bamtofastq_R1.fastq.gz SRR7159837/fastq/bamtofastq_R1.fastq.gz) <(zcat SRR7159836/fastq/bamtofastq_R2.fastq.gz SRR7159837/fastq/bamtofastq_R2.fastq.gz) GSM3140593/GSM3140593
```
I get the following error:
```
sh: -c: line 0: syntax error near unexpected token `('
sh: -c: line 0: `mixcr analyze 10x-sc-5gex -s hsa --rna --threads 4 --force-overwrite <(zcat SRR7159836/fastq/bamtofastq_R1.fastq.gz SRR7159837/fastq/bamtofastq_R1.fastq.gz) <(zcat /SRR7159836/fastq/bamtofastq_R2.fastq.gz SRR7159837/fastq/bamtofastq_R2.fastq.gz) GSM3140593/GSM3140593'
```
Please advise.

Thanks and good day.

**Comment by mizraelson** [comment, marked as answer]
I recommend renaming files (or creating links with different names), placing them in the same directory, and using a wildcard:

If you have files like:

```
SRR7159837_R1.fastq.gz
SRR7159837_R2.fastq.gz
SRR7159836_R1.fastq.gz
SRR7159836_R2.fastq.gz
```

You can run:

```
mixcr analyze 10x-sc-5gex \
    -s hsa \
    --rna \
    --threads 4 \
    --force-overwrite \
    SRR{{a}}_{{R}}.fastq.gz \
   GSM3140593/GSM3140593
```
